talisker/sentry.py

```diff
@@ -312,7 +313,7 @@ def ensure_talisker_config(kwargs):
     if kwargs.get('hook_libraries') is None:
         kwargs['hook_libraries'] = []
 
-    tags = kwargs.get('tags', {})
+    tags = kwargs.get('tags') or {}
```

Comment:
I prefer the original spelling of this, thanks. I prefer to avoid or for assignments, as it can backfire

Reply:
But in original spelling it crashes because at some point `tags` in kwargs is `None`

Reply:
Ah, yes, good catch, thanks!